mteb/tasks/BitextMining/multilingual/IN22ConvBitextMining.py

```diff
@@ -0,0 +1,151 @@
+    return {"hash": hash(text)}
+
+
+def check_uniques(example, uniques):
+    """Check if current hash is still in set of unique hashes and remove if true."""
+    if example["hash"] in uniques:
+        uniques.remove(example["hash"])
+        return True
+    else:
+        return False
+
+
+class IN22ConvBitextMining(AbsTaskBitextMining, CrosslingualTask):
+    metadata = TaskMetadata(
+        name="IN22ConvBitextMining",
+        dataset={
+            "path": "ai4bharat/IN22-Conv",
+            "revision": "189b3d92cfbde4ddb3514223a256eb78b1aca60f",
+            "trust_remote_code": True,
+        },
+        description="IN22-Conv is a n-way parallel conversation domain benchmark dataset for machine translation spanning English and 22 Indic languages.",
+        reference="https://huggingface.co/datasets/ai4bharat/IN22-Conv",
+        type="BitextMining",
+        category="s2s",
+        eval_splits=_SPLIT,
+        eval_langs=_LANGUAGES_MAPPING,
+        main_score="f1",
+        date=("2022-10-01", "2023-03-01"),
+        form=["spoken"],
+        domains=["Social", "Spoken", "Fiction"],
+        task_subtypes=[],
+        license="CC-BY-4.0",
+        socioeconomic_status="mixed",
+        annotations_creators="expert-annotated",
+        dialect=[],
+        text_creation="created",
+        bibtex_citation="""@article{gala2023indictrans,
+title={IndicTrans2: Towards High-Quality and Accessible Machine Translation Models for all 22 Scheduled Indian Languages},
+author={Jay Gala and Pranjal A Chitale and A K Raghavan and Varun Gumma and Sumanth Doddapaneni and Aswanth Kumar M and Janki Atul Nawale and Anupama Sujatha and Ratish Puduppully and Vivek Raghavan and Pratyush Kumar and Mitesh M Khapra and Raj Dabre and Anoop Kunchukuttan},
+journal={Transactions on Machine Learning Research},
+issn={2835-8856},
+year={2023},
+url={https://openreview.net/forum?id=vfT4YuzAYA},
+note={}
+}""",
+        n_samples={"conv": 1503},
+        avg_character_length={"conv": 54.3},
+    )
+
+    def load_data(self, **kwargs: Any) -> None:
+        """Load dataset from HuggingFace hub"""
+        if self.data_loaded:
+            return
+        self.dataset = {}
+        for lang in self.langs:
+            self.dataset[lang] = datasets.load_dataset(
+                name=lang,
+                **self.metadata_dict["dataset"],
+            )
+        self.dataset_transform()
+        self.data_loaded = True
+
+    def dataset_transform(self) -> None:
+        # Convert to standard format
+        for lang in self.langs:
+            lang1 = lang.split("-")[0]
+            lang2 = lang.split("-")[1]
+            for split in _SPLIT:
+                self.dataset[lang][split] = self.dataset[lang][split].rename_column(
+                    "sentence_" + lang1, "sentence1"
+                )
+                self.dataset[lang][split] = self.dataset[lang][split].rename_column(
+                    "sentence_" + lang2, "sentence2"
+                )
```

Comment:
```suggestion
            self.dataset[lang] = self.dataset[lang].rename_columns({
                    f"sentence_{lang1}": "sentence1",
                    f"sentence_{lang2}": "sentence2"
            )
```
You can remove the splits for loop and use `rename_columns()`, it will update column names of all splits.
Same thing for `map()` and `filter()`, see docs [here](https://huggingface.co/docs/datasets/process#process-multiple-splits).